tss-esapi/tests/integration_tests/context_tests/tpm_commands/integrity_collection_pcr_tests.rs

```diff
@@ -34,6 +33,12 @@ mod test_pcr_extend_reset {
         });
 
         // Needs to have the length of associated with the hashing algorithm
+        assert_eq!(read_pcr_selections.get_selections().len(), 2);
+        assert_eq!(
+            pcr_selection_list.get_selections(),
+            read_pcr_selections.get_selections()
+        );
```

Comment:
Are these `len` checks not redundant, given the other asserts? It might make debugging easier if the issue is a length discrepancy, I guess...

Reply:
Maybe I just wanted to be really sure there were no missmatch.